# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/VideoDecoder.h
# Commit: e903ac081227
# Full Hash: e903ac0812273c476426eba3462c4bb2977c4f2f
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 3
# Description:
#   Bug 1749047 - Move and deduplicate some utility functions to WebCodecUtils. r=chunmin
#   
#   They will be reused for the encoding side.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195699
# ==============================================================================

diff -r 029f5470314e -r e903ac081227 dom/media/webcodecs/VideoDecoder.h
--- a/dom/media/webcodecs/VideoDecoder.h	Fri Dec 22 21:39:04 2023 +0000
+++ b/dom/media/webcodecs/VideoDecoder.h	Fri Dec 22 21:39:05 2023 +0000
@@ -8,18 +8,14 @@
 #define mozilla_dom_VideoDecoder_h
 
 #include "js/TypeDecls.h"
-#include "mozilla/Attributes.h"
 #include "mozilla/ErrorResult.h"
-#include "mozilla/Maybe.h"
 #include "mozilla/RefPtr.h"
 #include "mozilla/UniquePtr.h"
 #include "mozilla/dom/BindingDeclarations.h"
 #include "mozilla/dom/DecoderTemplate.h"
 #include "mozilla/dom/DecoderTypes.h"
-#include "mozilla/dom/RootedDictionary.h"
 #include "mozilla/dom/VideoFrame.h"
 #include "nsCycleCollectionParticipant.h"
-#include "nsWrapperCache.h"
 
 class nsIGlobalObject;
 